doc/services/portability/posix/overview/index.rst

```diff
@@ -0,0 +1,131 @@
+POSIX-1.2017).
+
+..  figure:: posix.svg
+    :align: center
+    :alt: POSIX Support in Zephyr
+
+    POSIX support in Zephyr
+
+.. note::
+    This page does not document Zephyr's :ref:`POSIX architecture<Posix arch>`, which is used to
+    run Zephyr as a native application under the host operating system for prototyping,
+    test, and diagnostic purposes.
+
+With the POSIX support available in Zephyr, an existing POSIX conformant
+application can be ported to run on the Zephyr kernel, and therefore leverage
+Zephyr features and functionality. Additionally, a library designed to be
+POSIX conformant can be ported to Zephyr kernel based applications with no changes.
+
+The POSIX API is an increasingly popular OSAL (operating system abstraction layer) for IoT and
+embedded applications, as can be seen in Zephyr, AWS:FreeRTOS, TI-RTOS, and NuttX.
+
+Benefits of POSIX support in Zephyr include:
+
+- Offering a familiar API to non-embedded programmers, especially from Linux
+- Enabling reuse (portability) of existing libraries based on POSIX APIs
+- Providing an efficient API subset appropriate for small (MCU) embedded systems
+
+.. _posix_subprofiles:
+
+POSIX Subprofiles
+=================
+
+While Zephyr supports running multiple `threads <threads_v2>`_ (possibly in an `SMP <smp_arch>`_
+configuration), as well as `Virtual Memory and MMUs <x86_developer_guide>`_, Zephyr code and data
+normally share a common address space. The Zephyr kernel executable code and the application
+executable code are typically compiled into the same binary artifact. From that perspective, Zephyr
+apps can be seen as running in the context of a single process.
+
+While multi-purpose operating systems (OS) offer full POSIX conformance, Real-Time Operating
+Systems (RTOS) such as Zephyr typically serve a fixed-purpose, have limited hardware resources,
+and experience limited user interaction. In such systems, full POSIX conformance can be
+impractical and unnecessary.
+
+For that reason, POSIX defined the following :ref:`Application Environment Profiles (AEP)<posix_aep>`
+as part of `IEEE 1003.13-2003`_ (also known as POSIX.13-2003).
+
+* Minimal Realtime System Profile (:ref:`PSE51 <posix_aep_pse51>`)
+* Realtime Controller System Profile (:ref:`PSE52 <posix_aep_pse52>`)
+* Dedicated Realtime System Profile (:ref:`PSE53 <posix_aep_pse53>`)
+* Multi-Purpose Realtime System (PSE54)
+
+POSIX.13-2003 AEP were formalized in 2003 via "Units of Functionality" but the specification is now
+inactive (for reference only). Nevertheless, the intent is still captured as part of POSIX-1.2017
+via :ref:`Options<posix_options>` and :ref:`Option Groups<posix_option_groups>`, in Appendix E.
+
+.. _posix_apps:
+
+POSIX Applications in Zephyr
+============================
+
+A POSIX app in Zephyr is :ref:`built like any other app<application>`; it requires the usual
+`prj.conf` and `CMakeLists.txt`, and then it is :ref:`built the usual way<samples-and-demos>`.
```

Comment:
i don't think the second reference makes sense (and it's redundant)
also a minor formatting change on files.

```suggestion
A POSIX app in Zephyr is :ref:`built like any other app<application>` and therefore requires the usual
:file:`prj.conf` and :file:`CMakeLists.txt` files.
```

Reply:
Fixed!